### ISSUE TITLE
Draw leaf stems of polytomies in light gray too

### ISSUE THREAD

**Issue by jaredkhan** [OneZoom/OZtree#909]
Polytomies in the default Spiral view are drawn in light grey but the stems of the leafs are drawn in dark grey. This includes midnodes that are children of polytomies. I wonder if it'd be clearer of those stems reflected the polytomy-ness of the parent so that dark gray always indicates genuine phylogeny knowledge.

<img width="834" alt="image" src="https://github.com/user-attachments/assets/9ba41959-5972-4ecd-84ee-b01e30af3063">

Mockup 
<img width="903" alt="image" src="https://github.com/user-attachments/assets/ec07bc05-1cc5-45b9-ae67-614a2fc1231d">

**Comment by jaredkhan** [comment]
Think it also has the benefit of being visually a bit cleaner because there's not so much gray-to-darkgray boundaries

**Comment by davidebbo** [comment]
Shouldn't the leaf connection to KI89A also be light gray?

**Comment by jaredkhan** [comment]
@davidebbo maybe yeah, was just a scrappy mockup, but I guess part of the point is that it’s not obvious at the moment whether that is a genuine sibling of the polytomy or part of it.

 But maybe if I thought harder about it it would be clear haha

**Comment by davidebbo** [comment]
Yes, it definitely part of the polytomy. Otherwise, the polytomy would only have 2 leaves, and it wouldn't be a polytomy :)

Here is the newick chunk for reference:
`(obligately_oligotrophic_bacterium_KI89C_ott4790582,marine_gamma_proteobacterium_HTCC2089_ott4790583,K189A_clade_bacterium_ott7035165)KI89A_clade_ott5246876`
